bin/rebuild

```diff
@@ -73,7 +73,7 @@ save_build_env() {
   echo "Saving environment information in ${filename}"
 
   # saving information in environment name
-  echo "#environment_name: ${LSST_CONDA_ENV_NAME}" > "${filename}"
+  echo "# environment_name: ${LSST_CONDA_ENV_NAME}" > "${filename}"
```

Comment:
I think this is going to affect the `-f 2` in `envconfig` line 128.

Reply:
right, better to avoid this change